### PR TITLE
Fix .Net Setup app and make SDK work with plain text private key

### PR DESCRIPTION
**Pull request by ppatidar2021**

### CODE REVIEW

File: BitPay/BitPay.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Microsoft.Extensions.Configuration;
+using Org.BouncyCastle.Math;
```
Comment (anbugal) [inline BitPay/BitPay.cs:22]: What is this one used for?

**Review by anbugal — CHANGES_REQUESTED** [review]

File: BitPay/BitPay.cs (hunk 1 above)
Comment (nsoni3) [reply BitPay/BitPay.cs:22]: To use the private key as a BigInterger.

Comment (nsoni3) [reply BitPay/BitPay.cs:22]: > What is this one used for?

To use the private key as a BitInteger.

File: BitPayXUnitTest/BitPayTests.cs [hunk 2]
```diff
@@ -43,7 +44,9 @@ public Tests()
         {
             // JSON minified with the BitPay configuration as in the required configuration file
             // and parsed into a IConfiguration object
-            var json = "{\"BitPayConfiguration\":{\"Environment\":\"Test\",\"EnvConfig\":{\"Test\":{\"PrivateKeyPath\":\"sec/bitpay_test_private.key\",\"ApiTokens\":{\"merchant\":\"A4qqz5JXoK5TMi3hD8EfKNHJB2ybLgdYRkbZwZ5M9ZgT\",\"payout\":\"G4pfTiUU7967YJs7Z7n8e2SuQPa2abDTgFrjFB5ZFZsT\"}},\"Prod\":{\"PrivateKeyPath\":\"\",\"ApiTokens\":{\"merchant\":\"\"}}}}}";
+            //var json = "{\"BitPayConfiguration\":{\"Environment\":\"Test\",\"EnvConfig\":{\"Test\":{\"PrivateKeyPath\":\"sec/bitpay_test_private.key\",\"ApiTokens\":{\"merchant\":\"A4qqz5JXoK5TMi3hD8EfKNHJB2ybLgdYRkbZwZ5M9ZgT\",\"payout\":\"G4pfTiUU7967YJs7Z7n8e2SuQPa2abDTgFrjFB5ZFZsT\"}},\"Prod\":{\"PrivateKeyPath\":\"\",\"ApiTokens\":{\"merchant\":\"\"}}}}}";
+            
+            var json = File.ReadAllText(@"BitPay.config.json");
```
Comment (anbugal) [inline BitPayXUnitTest/BitPayTests.cs:49]: it is easier to handle the above version of the config

File: BitPay/BitPay.cs (hunk 1 above)
Comment (anbugal) [reply BitPay/BitPay.cs:22]: But I don't see where this is being used in this class

**Review by anbugal — CHANGES_REQUESTED** [review]

**Pull request merged**